<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.1 (Python 3.7.4 on win32)" generated="20200525 15:48:43.385" rpa="false">
<suite id="s1" name="Api2" source="D:\PycharmProjects\rf_auto\API_TEST\Tasks\api2.robot">
<test id="s1-t1" name="POST Requests">
<kw name="Create Session" library="RequestsLibrary">
<doc>Create Session: create a HTTP session to a server</doc>
<arguments>
<arg>api</arg>
<arg>${domain}</arg>
</arguments>
<msg timestamp="20200525 15:48:43.499" level="FAIL">Variable '${domain}' not found.</msg>
<status status="FAIL" starttime="20200525 15:48:43.499" endtime="20200525 15:48:43.499"></status>
</kw>
<doc>POST请求json</doc>
<status status="FAIL" starttime="20200525 15:48:43.498" endtime="20200525 15:48:43.499" critical="yes">Variable '${domain}' not found.</status>
</test>
<status status="FAIL" starttime="20200525 15:48:43.387" endtime="20200525 15:48:43.500"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Api2">Api2</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20200525 15:48:43.498" level="ERROR">Error in file 'D:\PycharmProjects\rf_auto\API_TEST\Tasks\api2.robot' on line 4: Test library 'test_env.py' does not exist.</msg>
</errors>
</robot>
